spec/cassettes/Octokit_Client_Organizations/_organization_audit_log/returns_audit_log_for_an_organization.json

```diff
@@ -0,0 +1 @@
+{"http_interactions":[{"request":{"method":"get","uri":"https://api.github.com/orgs/api-playground/audit-log?include=all&phrase=action:org.add_member%20created:%3E2022-08-29","body":{"encoding":"US-ASCII","base64_string":""},"headers":{"Accept":["application/vnd.github.v3+json"],"User-Agent":["Octokit Ruby Gem 5.4.0"],"Content-Type":["application/json"],"Authorization":["token <<ACCESS_TOKEN>>"],"Accept-Encoding":["gzip;q=1.0,deflate;q=0.6,identity;q=0.3"]}},"response":{"status":{"code":200,"message":"OK"},"headers":{"Server":["GitHub.com"],"Date":["Thu, 08 Sep 2022 09:46:38 GMT"],"Content-Type":["application/json; charset=utf-8"],"Transfer-Encoding":["chunked"],"Cache-Control":["private, max-age=60, s-maxage=60"],"Vary":["Accept, Authorization, Cookie, X-GitHub-OTP","Accept-Encoding, Accept, X-Requested-With"],"Etag":["W/\"b3452f345082e67e0b56ebea9a8eb8a079d51bebf854b03ad5087fcdc011f276\""],"X-Oauth-Scopes":["admin:org, repo:org, write:org"],"X-Accepted-Oauth-Scopes":["admin:org, read:org, write:org"],"Github-Authentication-Token-Expiration":["2022-09-08 22:00:00 UTC"],"X-Github-Media-Type":["github.v3; format=json"],"X-Ratelimit-Limit":["5000"],"X-Ratelimit-Remaining":["4991"],"X-Ratelimit-Reset":["1662630565"],"X-Ratelimit-Used":["9"],"X-Ratelimit-Resource":["core"],"Access-Control-Expose-Headers":["ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"],"Access-Control-Allow-Origin":["*"],"Strict-Transport-Security":["max-age=31536000; includeSubdomains; preload"],"X-Frame-Options":["deny"],"X-Content-Type-Options":["nosniff"],"X-Xss-Protection":["0"],"Referrer-Policy":["origin-when-cross-origin, strict-origin-when-cross-origin"],"Content-Security-Policy":["default-src 'none'"],"X-Github-Request-Id":["4133:295B:16ADE1C:17110CD:6319B9FE"]},"body":{"encoding":"ASCII-8BIT","base64_string":"WwogICAgewogICAgICAgICJhY3RvciI6ICJvY3Rva2l0IiwKICAgICAgICAiY3JlYXRlZF9hdCI6IDE2NjE4MzY2MTA2NjIsCiAgICAgICAgIm9yZyI6ICJjb2RlZm9yYW1lcmljYSIsCiAgICAgICAgInBlcm1pc3Npb24iOiAicmVhZCIsCiAgICAgICAgIkB0aW1lc3RhbXAiOiAxNjYxODM2NjEwNjYyLAogICAgICAgICJidXNpbmVzcyI6ICJjb2RlZm9yYW1lcmljYSIsCiAgICAgICAgInVzZXIiOiAibmV3LW9jdG9raXQiLAogICAgICAgICJhY3Rpb24iOiAib3JnLmFkZF9tZW1iZXIiLAogICAgICAgICJfZG9jdW1lbnRfaWQiOiAiREtqVWR2aVRBcTViU0cxVEdtOGxjQSIsCiAgICAgICAgIm9wZXJhdGlvbl90eXBlIjogImNyZWF0ZSIKICAgIH0KXQ=="}},"recorded_at":"Thu, 08 Sep 2022 09:46:38 GMT"}],"recorded_with":"VCR 6.1.0"}
```

Comment:
What did you have to do to get this cassette to generate successfully? I've been trying to get this working in my local development environment for a while and I've been struggling.

Reply:
What issues are you running into? 

What I did is the following:
- Get a token for the test organization
- Make sure recordings are enabled to generate the cassette.
- Trigger the test to save the cassette.
- Manually edit the cassette to ensure no private data is included. I ran it across a Philips test organization... not really ideal, but I didn't know how to otherwise do it.
	- While manually editing the cassette, I had to decode the base64 ASCI II string to edit the values, and encode it back to base64 ASCI II string. It's a bit clunky, but I suppose it works?

Reply:
What did you have to do to make sure that recording was enabled?

Reply:
See https://github.com/octokit/octokit.rb/pull/1480#issuecomment-1245528568